.travis.yml

```diff
@@ -11,4 +11,9 @@ jdk:
 install: true
 
 script:
-  - mvn verify
\ No newline at end of file
+  - docker network create wdfn
+  - docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$TRAVIS_BUILD_DIR":/usr/src/mymaven -w /usr/src/mymaven --network=wdfn -e TESTING_DATABASE_PORT=5432 -e TESTING_DATABASE_ADDRESS=wdfnDatabase -e TESTING_DATABASE_NETWORK=wdfn maven:3.6.0-jdk-11 mvn clean verify
```

Comment:
Can we split this command into multiple lines?  This is only for readability's sake - if it's inconvenient please disregard.